docs/pdr_avhw.md

```diff
@@ -0,0 +1,619 @@
+
+Recovery assist still needs concept determined to create design. 
+
+### Communications Card
+
+Implementation of this will rely heavily on Dan’s amplifier designs and overall implementation, however major portions have already been decided such as the SDR so work can begin. Block diagram and basic interaction outside of SDR TBD.
+
+## Timeline
+
+### MFC Card Development
+
+3 weeks to complete schematic
+
+3 weeks to complete PCB
+
+3 weeks for manufacturing
+
+### Payload Card development
+
+2 weeks to complete template schematic
+
+1 week to complete customized payload card schematic
+
+3 weeks to complete PCB
+
+3 weeks for manufacturing
+
+### Development plan
+I believe this is literally waterfall design. There will be some jumping around between cards as plans are finalized however see below for the approximate development path. The goal will be to simultaneously work on two project points at one point to near completion ( i.e. enough to put down to pick up the next two points. Currently the Backplane schematic is “complete” and work has started on the Backplane PCB and MFC Card schematic. 
+
+![timeline](https://user-images.githubusercontent.com/12124823/93687139-a3d6bc00-fa78-11ea-8a13-1b4806cb1e51.png)
+
+PCB assembly will potentially occur during this development process based on how the card design finalizes and the needs of the software development side of things. However assembly and test of PCBAs is not included in this development schedule. Current expectation is that boards will be ordered and assembled after CDR.
+
+## Cost:
+* Backplane - $150 ea
+* Main Flight Computer Card - $300 ea
+* Avionics Sensor Card - $200 ea
+* Altitude Control/Dynamics Sensor Card - $200 ea
+* Communications Card - $300 ea
+* Power Supply Card - $200 ea
+* Balloon Board - $150 ea
+Total: $1500 per set
+
+Please note, this is a very rough estimate. The goal is to drive costs lower while maintaining high levels of features (I promise I'm not on the marketing team)
+
+## Card Specific Information
+
+### Backplane
+
+#### Theory of Operation
+This board acts as an interconnect between all payload cards and provides routing for all power and data connections. There is one dedicated card slot for the Main Flight Computer, Communications card and one for the Power Supply Card. In addition there are four payload card slots for various cards. Currently for Nucleus there is an Avionics Sensor Card, Altitude Control/Dynamics Sensor Card for payload cards
+
+Backplane Concepting  - Ethernet Sheet
+
+#### Electrical Interface
+
+##### Connectors
+There is one 40 pin Molex MilliGrid vertical receptacle, one 34 pin Molex MilliGrid vertical receptacle and six 28 pin Molex MilliGrid vertical receptacles providing the interfaces between the various avionics cards. There is also a Molex Microfit header and RJ45 connector for ground station interface and a 10 pin JST header for connection to the balloon board.
+
+###### Main Flight Computer Interface
```

Comment:
If there's only 1 line of text after a heading, it might work better as a table?

Reply:
i switched it to bold text, with the further sub headings being italics